lib/Sema/CodeSynthesis.cpp

```diff
@@ -849,14 +849,14 @@ bool AreAllStoredPropertiesDefaultInitableRequest::evaluate(
   decl->collectPropertiesInitializableByInitAccessors(
       initializedViaInitAccessor);
 
-  for (auto member : decl->getImplementationContext()->getMembers()) {
```

Comment:
I think all you need is to change this call to `getMembers()` to `getAllMembers()`.